### ISSUE TITLE
feat(knowledge-view): be able to view all sightings of a specific Observable

### ISSUE THREAD

**Issue by SouadHadjiat** [LuatixHQ/opencti#8398]
## Description

Sightings created from knowledge view of a report are not found on the indicator. See Reproducible Steps. There might be other use cases where created sightings on indicator are not displayed on the indicator.

## Environment

1. OS (where OpenCTI server runs): { e.g. Mac OS 10, Windows 10, Ubuntu 16.4, etc. }
2. OpenCTI version: 6.3+ (and latest 6.2)
3. OpenCTI client: frontend
4. Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Go to a report with a indicator and a system (or create a new one and add 1 indicator and 1 system)
2. From the knowledge view, create a sighting between the indicator and the system : sighting is created.
3. Go to the indicator > sightings tab => it's empty (the created sighting is not displayed).

**Comment by nino-filigran** [comment]
I created a report in which I added 2 IOCs & a system: 
- 1 IOC -> sighted at -> System: 
   - IOC: no sighting 
   - System: 3 sightings (2 added due inference rules) of the IOC
- 1 system -> sighted at -> IOC 
   - system: no sighting
   - IOC: sighting of the system. 

Not sure if it's a bug, I'll come back to you.

**Comment by nino-filigran** [comment]
@SouadHadjiat I've switched this to feature. There are few reasons for this: 
- usually you would create a relation Observable -> sighted in/at -> system/org...  Therefore seing the sighting in the "target" entity makes sense.
- however it's important to visualize all entities in which your observables has been sighted: even if we could use the screen as-is, the UI would be misleading. Therefore, we need to properly think about how to display this information in the application. And this has not be been implemented so far.

**Comment by MaxwellDPS** [comment]
Yeah 2nd-ing this one - Summarizing sightings is a largely meaningless metric
